provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -1,14 +1,37 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using DotNetty.Buffers;
+using DotNetty.Codecs.Mqtt;
+using DotNetty.Handlers.Tls;
+using DotNetty.Transport.Bootstrapping;
+using DotNetty.Transport.Channels;
+using DotNetty.Transport.Channels.Sockets;
+using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+using Microsoft.Azure.Devices.Provisioning.Client.Transport.Models;
+using Microsoft.Azure.Devices.Shared;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Net.WebSockets;
+using System.Security.Cryptography.X509Certificates;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.Devices.Shared;
 
 namespace Microsoft.Azure.Devices.Provisioning.Client.Transport
 {
     public class ProvisioningTransportHandlerMqtt : ProvisioningTransportHandler
     {
+        private static MultithreadEventLoopGroup s_eventLoopGroup = new MultithreadEventLoopGroup();
+        private const int MaxMessageSize = 256 * 1024;
+        private const int MqttTcpPort = 8883;
+
+        private const string WsMqttSubprotocol = "mqtt";
+        private const string WsScheme = "wss";
+        private const int WsPort = 443;
```

Comment:
we have these (similar) define in WebSocketConstants.  Check if we can consolidate.

Reply:
The file containing these defines, for some reason, mixes AMQP and MQTT. Leaving this as a TODO.